Foundation/AffineTransform.swift

```diff
@@ -476,7 +476,7 @@ open class NSAffineTransform : NSObject, NSCopying, NSSecureCoding {
     }
 }
 
-extension AffineTransform : _ObjectTypeBridgeable {
+extension AffineTransform : _ObjectiveCBridgeable {
```

Comment:
The main change is removing the parallel type `_ObjectTypeBridgeable` and replacing it with the newly-exposed-on-Linux `_ObjectiveCBridgeable`.